tools/cpp/CROSSTOOL.tpl

```diff
@@ -510,6 +502,39 @@ toolchain {
     }
   }
 
+  feature {
+    name: "msvc_compile_env"
+    enabled: true
```

Comment:
As mentioned in a comment for other PR, `enabled: true` can be disabled, which is probably not what we want (user will get compile/build error without these env vars). I can make `msvc_env` "implies" `msvc_compile_env` and `msvc_link_env` so that I don't have to add these features to all `action_config`s manually. Sounds good?

Reply:
Sounds great!

Reply:
Done.